_extras/customization.md

```diff
@@ -140,16 +148,69 @@ so that they show what you're actually planning to teach and when.  These
 files are located in the appropriate workshop folder (`dc`, `lc` or `swc`)
 inside the `_includes` folder.
 
-## Home Page: Setup
+## Home Page: Setup Instructions
 
-You may have to edit the `setup.html` located
-in the `dc`, `lc` or `swc` folders
-inside the `_includes` folder.
-Make sure you delete the sections for the
-software and data
-you will **not** be using in your workshop,
-so that learners don't spend time installing
-software they don't need.
+If you need assistance with customizing the setup instructions for your website,
+feel free to ask your questions in the Carpentries
+[Instructors Slack channel](https://swcarpentry.slack.com/archives/C08BVNU00)
```

Comment:
noticed `swcarpentry` here. If desired, Slack org name can be changed to `carpentries`:
https://slack.com/help/articles/201663443-Change-your-workspace-or-org-name-and-URL

Reply:
yes, we have this in the works but there are some complications we need to resolve before we can finally move forward on this renaming.